src/api/Shrooms.Domain/Services/Wall/Posts/PostService.cs

```diff
@@ -144,6 +145,28 @@ public void EditPost(EditPostDTO editPostDto)
                 _uow.SaveChanges(editPostDto.UserId);
             }
         }
+        public string GetPostTitle(int postId)
```

Comment:
I changed it to `GetPostBody()`, because `GetPost()` for me sounds more like you are returning an object (with comments and other properties) rather than the string.